src/components/AddNewBookForm/AddNewBookForm.js

```diff
@@ -4,15 +4,19 @@ import {connect} from 'react-redux';
 import {compose} from 'recompose';
 import {addNewUserBook} from '../../redux/actions/addNewUserBook';
 import {withFirebase} from '../Firebase';
+import {uploadPictureToFirebase} from '../../helper/storageHelper';
 import Button from '../Button/Button';
+import TitleForm from './TitleForm';
 
-const AddNewBookForm = props => {
-  const {addNewUserBook, author, title, rating, firebase} = props;
+const AddNewBookFormBase = props => {
+  const {addNewUserBook, author, title, rating, firebase, files} = props;
   const [description, setDescription] = useState('');
   const [location, setLocation] = useState({
     lat: 0,
     lon: 0
   });
+  const [type, setType] = useState('lend');
+  var imageUrls = [];
```

Comment:
Maybe change this to ```let```

Reply:
No need to change I think. `const` only affects the `List` itself, not objects in it.